File: C++/15.cpp
#include <iostream>

using namespace std;

int main()
{

    int size;

    cout << "Enter the number of elements: ";
    cin >> size;

    int list[size];

    cout << "Enter elements of the lis: " << endl;

    for (int i = 0; i < size; i++)
        cin >> list[i];

    int minElement = list[0];

    for (int i = 0; i < size; i++)
    {
        if (list[i] < minElement)
            minElement = list[i];
    }

    cout << "Min element of the list: " << minElement;
}

File: Python/15.py
myList = []

numElements = int(input("How many elements would you like to add? "))

for i in range(numElements):
    element = int(input("Enter an element: "))
    myList.append(element)

minElement = myList[0]

for i in range(numElements):
    if myList[i] < minElement:
        maxElement = myList[i]
print(minElement)
